static/index.html

```diff
@@ -6,9 +6,15 @@
     <title>Index</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="stylesheet" type="text/css" media="screen" href="css/main.css" />
-    <script src="js/main.js"></script>
+
 </head>
 <body>
-    
+    <h1>Connected Modules</h1>
+    <p id="cTable">
+    </p>
+    <h1>Disconnected Modules</h1>
+    <p id="dcTable"></p>
+    <button onclick="refresh()">Refresh</button>
+    <script src="js/main.js"></script>
```

Comment:
Do not include the JS file like this. It should be in the head tag like it was originally.

Reply:
Yes that was temporary, script was trying to run before elements were added, forgot to fix.  Added a begin() function that adds elements to page in the main.js file.